SG14/transcode.h

```diff
@@ -103,6 +103,7 @@ sg14::rsa_decryptor<OutIt>::rsa_decryptor(OutIt&& dest_, int m_, int n_)
 template<typename OutIt>
 sg14::rsa_decryptor<OutIt>& sg14::rsa_decryptor<OutIt>::operator=(const char& element)
 {
-	OutIt::operator=(element - m);
+	OutIt::operator=(static_cast<char>(element - m));
```

Comment:
I'll merge this diff right now, even though I would still like @hatcat to think about moving this entire header into a non-master branch and/or deleting it.

Reply:
thx, just trying to get it to build on vs2017